### PR TITLE
fix doc of erf,CrossEntropyLoss,cross_entropy,erf,rank,adaptive_max_pool1d,pixel_shuffle,kron,mm

### PR DESCRIPTION
**Pull request by xmy0916**
# PR types
Bug fixes

# PR changes
Docs

# Describe
fix doc of erf,CrossEntropyLoss,cross_entropy,erf,rank,adaptive_max_pool1d,pixel_shuffle,kron,mm

### CODE REVIEW

File: doc/fluid/api_cn/layers_cn/rank_cn.rst [hunk 1]
```diff
@@ -5,28 +5,24 @@ rank
 
 .. py:function::  paddle.fluid.layers.rank(input)
```
Comment (TCChenlong) [inline doc/fluid/api_cn/layers_cn/rank_cn.rst]: 这里改成 paddle.rank

File: doc/fluid/api_cn/nn_cn/loss_cn/CrossEntropyLoss_cn.rst [hunk 2]
```diff
@@ -41,7 +41,6 @@ CrossEntropyLoss
 
             import paddle
             import numpy as np
-            paddle.disable_static()
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/loss_cn/CrossEntropyLoss_cn.rst:44]: 第六行 改成 paddle.nn.CrossEntropyLoss

File: doc/fluid/api_cn/tensor_cn/kron_cn.rst [hunk 3]
```diff
@@ -5,10 +5,6 @@ kron
 
 .. py:function:: paddle.tensor.kron(x, y, out=None, name=None)
 
-:alias_main: paddle.kron
```
Comment (TCChenlong) [inline doc/fluid/api_cn/tensor_cn/kron_cn.rst:8]: 第六行改成 paddle.kron

File: doc/paddle/api/paddle/fluid/layers/erf_cn.rst [hunk 4]
```diff
@@ -3,7 +3,7 @@
 erf
 -------------------------------
 
-.. py:function:: paddle.fluid.layers.erf(x)
+.. py:function:: paddle.erf(x)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/erf_cn.rst]: 少了 name 参数

File: doc/paddle/api/paddle/fluid/layers/erf_cn.rst [hunk 5]
```diff
@@ -15,58 +15,18 @@ erf
     out = \frac{2}{\sqrt{\pi}} \int_{0}^{x}e^{- \eta^{2}}d\eta
 
 参数：
-  - **x** (Variable) - Erf Op 的输入，多维 Tensor 或 LoDTensor，数据类型为 float16, float32 或 float64。
+  - **x** (Tensor) - Erf Op 的输入，多维 Tensor 或 LoDTensor，数据类型为 float16, float32 或 float64。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/erf_cn.rst:18]: 少了 name 参数

File: doc/paddle/api/paddle/tensor/math/mm_cn.rst [hunk 6]
```diff
@@ -15,16 +15,14 @@ mm
...
-    - **input** (Variable) : 输入变量，类型为 Tensor 或 LoDTensor。
-    - **mat2** (Variable) : 输入变量，类型为 Tensor 或 LoDTensor。
-    - **out** (Variable, 可选) – 指定存储运算结果的Tensor。如果设置为None或者不设置，将创建新的Tensor存储运算结果，默认值为None。
+    - **input** (Tensor) : 输入变量，类型为 Tensor 或 LoDTensor。
+    - **mat2** (Tensor) : 输入变量，类型为 Tensor 或 LoDTensor。
+    - **out** (Tensor, 可选) – 指定存储运算结果的Tensor。如果设置为None或者不设置，将创建新的Tensor存储运算结果，默认值为None。
     - **name** （str，可选）- 具体用法请参见 :ref:`api_guide_Name` ，一般无需设置，默认值为None。
 
 返回：
     - Variable (Tensor / LoDTensor)，矩阵相乘后的结果。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/tensor/math/mm_cn.rst]: 返回：
    Tensor，矩阵相乘后的结果。

File: doc/paddle/api/paddle/tensor/math/kron_cn.rst [hunk 7]
```diff
@@ -16,8 +16,8 @@ Kronecker Product 算子。
 
 
 这个 OP 预设两个张量 $X$ 和 $Y$ 的秩 (rank) 相同，如有必要，将会在秩较小的张量的形状前面补
-上 1。令 $X$ 的形状是 [$r_0$, $r_1$, ..., $r_N$]，$Y$ 的形状是 
-[$s_0$, $s_1$, ..., $s_N$]，那么输出张量的形状是 
+上 1。令 $X$ 的形状是 [$r_0$, $r_1$, ..., $r_N$]，$Y$ 的形状是
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/tensor/math/kron_cn.rst:19]: 第六行改成 paddle.kron

File: doc/paddle/api/paddle/nn/layer/loss/CrossEntropyLoss_cn.rst [hunk 8]
```diff
@@ -41,7 +41,6 @@ CrossEntropyLoss
 
             import paddle
             import numpy as np
-            paddle.disable_static()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/layer/loss/CrossEntropyLoss_cn.rst:44]: 第六行改成 paddle.nn.CrossEntropyLoss
np.random.random 可以改为 paddle.uniform

File: doc/paddle/api/paddle/nn/functional/vision/pixel_shuffle_cn.rst [hunk 9]
```diff
@@ -1,9 +1,9 @@
-.. _cn_api_fluid_layers_pixel_shuffle:
+.. _cn_api_paddle_functional_pixel_shuffle:
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/vision/pixel_shuffle_cn.rst]: 这个文件我修复了 你可以忽略掉

File: doc/paddle/api/paddle/nn/functional/loss/cross_entropy_cn.rst [hunk 10]
```diff
@@ -1,9 +1,9 @@
-.. _cn_api_fluid_layers_cross_entropy:
+.. _cn_api_paddle_functional_cross_entropy:
 
 cross_entropy
 -------------------------------
 
-.. py:function:: paddle.fluid.layers.cross_entropy(input, label, soft_label=False, ignore_index=-100)
+.. py:function:: paddle.paddle.nn.functional.loss.cross_entropy(input, label, weight=None, ignore_index=-100, reduction="mean", soft_lable=False, axis=-1, name=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/loss/cross_entropy_cn.rst]: paddle.paddle.nn.functional.loss.cross_entropy多了一个paddle

File: doc/paddle/api/paddle/nn/functional/loss/cross_entropy_cn.rst [hunk 11]
```diff
@@ -21,22 +21,24 @@ cross_entropy
...
 ..  code-block:: python
 
-        import paddle.fluid as fluid
-        class_num = 7
-        x = fluid.layers.data(name='x', shape=[3, 10], dtype='float32')
-        label = fluid.layers.data(name='label', shape=[1], dtype='int64')
-        predict = fluid.layers.fc(input=x, size=class_num, act='softmax')
-        cost = fluid.layers.cross_entropy(input=predict, label=label)
+        import paddle
+        input_data = np.random.random([5, 100]).astype("float64")
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/loss/cross_entropy_cn.rst]: 可以直接用 paddle.rand

File: doc/paddle/api/paddle/nn/functional/pooling/adaptive_max_pool1d_cn.rst [hunk 12]
```diff
@@ -48,7 +48,6 @@ adaptive_max_pool1d
         import paddle
         import paddle.nn.functional as F
         import numpy as np
-        paddle.disable_static()
 
         data = paddle.to_tensor(np.random.uniform(-1, 1, [1, 3, 32]).astype(np.float32))
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/pooling/adaptive_max_pool1d_cn.rst]: paddle.uniform 就不需要 numpy了

File: doc/paddle/api/paddle/nn/functional/pooling/max_pool1d_cn.rst [hunk 13]
```diff
@@ -46,7 +46,6 @@ max_pool1d
         import paddle
         import paddle.nn.functional as F
         import numpy as np
-        paddle.disable_static()
 
         data = paddle.to_tensor(np.random.uniform(-1, 1, [1, 3, 32]).astype(np.float32))
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/pooling/max_pool1d_cn.rst]: paddle.uniform

File: doc/paddle/api/paddle/nn/layer/pooling/AdaptiveMaxPool1D_cn.rst [hunk 14]
```diff
@@ -59,7 +59,6 @@ AdaptiveMaxPool1D
         import paddle
         import paddle.nn as nn
         import numpy as np
-        paddle.disable_static()
         
         data = paddle.to_tensor(np.random.uniform(-1, 1, [1, 3, 32]).astype(np.float32))
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/layer/pooling/AdaptiveMaxPool1D_cn.rst:63]: paddle.uniform

File: doc/paddle/api/paddle/nn/functional/loss/cross_entropy_cn.rst [hunk 15]
```diff
@@ -1,9 +1,9 @@
-.. _cn_api_fluid_layers_cross_entropy:
+.. _cn_api_paddle_functional_cross_entropy:
 
 cross_entropy
 -------------------------------
 
-.. py:function:: paddle.fluid.layers.cross_entropy(input, label, soft_label=False, ignore_index=-100)
+.. py:function:: paddle.nn.functional.loss.cross_entropy(input, label, weight=None, ignore_index=-100, reduction="mean", soft_lable=False, axis=-1, name=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/loss/cross_entropy_cn.rst]: paddle.nn.functional.loss.cross_entropy -> paddle.nn.functional.cross_entropy

**Review by TCChenlong — APPROVED** [review]
LGTM
